docs/articles_en/openvino-workflow/model-preparation/convert-model-jax.rst

```diff
@@ -0,0 +1,177 @@
+       def __call__(self, x):
+           return nn.Dense(features=self.features)(x)
+
+   module = SimpleModule(features=4)
+
+   # create example_input used for training
+   example_input = jnp.ones((2, 3))
+
+   # prepare parameters to initialize the module
+   # they can be also loaded using pickle, flax.serialization
+   key = jax.random.PRNGKey(0)
+   params = module.init(key, example_input)
+   module = module.bind(params)
+
+   ov_model = ov.convert_model(module, example_input=example_input)
+
+
+When using ``flax.linen.Module`` as an input model, ``openvino.convert_model`` requires the
+``example_input`` parameter to be specified. Internally, it triggers the model tracing during
+the model conversion process, using the capabilities  of the ``jax.make_jaxpr`` function.
+
+The ``__call__`` method of ``flax.linen.Module`` object can also have extra custom flags
+(like ``training``) in the input signature. In this case, it is required to create a helper function
+which has an input signature without any extra custom flags and parameters not related to input data.
+Here is an example of handling such case below:
+
+.. code-block:: py
+   :force:
+
+   import jax
+   import jax.numpy as jnp
+   import openvino as ov
+   from flax import linen as nn
+   from flax.core import freeze, unfreeze
+
+   class SimpleModuleWithExtraFlag(nn.Module):
+       features: int
+
+       @nn.compact
+       def __call__(self, x, training):
+           x = nn.Dense(self.features)(x)
+           x = nn.BatchNorm(use_running_average=not training)(x)
+           return x
+
+
+   # 1. Initialize the model
+   module = SimpleModuleWithExtraFlag(features=10)
+   key = jax.random.PRNGKey(0)
+   input_data = jnp.ones((4, 5))  # Batch of 4 samples, each with 5 features
+   params = module.init(key, input_data, training=False)
+
+   # 2. Create helper function with only input data parameter
+   def helper_function(x):
+       return module.apply(params, x, training=False)
+
+   # 3. Trace the helper function
+   jaxpr = jax.make_jaxpr(helper_function)(input_data)
+
+   # 4. Convert to OpenVINO
+   ov_model = ov.convert_model(jaxpr)
+
+
+.. note::
+
+   In the examples above the ``openvino.save_model`` function is not used because there are no
+   JAX-specific details regarding the usage of this function. In all examples, the converted
+   OpenVINO model can be saved to IR by calling ``ov.save_model(ov_model, 'model.xml')`` as usual.
+
+Exporting a JAX/Flax Model to TensorFlow SavedModel Format
+##########################################################
+
+An alternative method of converting JAX/Flax models is exporting a JAX/Flax model to TensorFlow SavedModel format
+with ``jax.experimental.jax2tf.convert`` first and then converting the resulting SavedModel directory to OpenVINO Model
+with ``openvino.convert_model``. It can be considered as a backup solution if a model cannot be
+converted directly from JAX/Flax to OpenVINO as described in the chapters above.
```

Comment:
```suggestion
with ``openvino.convert_model``. It can be considered a backup solution, if a model cannot be
converted directly, as described previously.
```